src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -278,9 +278,9 @@ private ImmutableArray<IOperation> GetIOperationChildren(BoundNode boundNode)
             return builder.ToImmutableAndFree();
         }
 
-        private IVariableDeclarationOperation CreateVariableDeclaration(BoundLocalDeclaration boundNode)
+        private IVariableDeclaratorOperation CreateVariableDeclaration(BoundLocalDeclaration boundNode)
```

Comment:
method name mismatch?

Reply:
Fixed.

---
In reply to: [148083892](https://github.com/dotnet/roslyn/pull/22842#discussion_r148083892) [](ancestors = 148083892)